src/components/Home/Today1Hourly/SvgBackground/SvgBackground.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { CombinedState } from '../../../../store/index-redux';
+import { TempUnits } from '../../../../store/tempUnits-redux';
+import CreateLine from './CreateLine/CreateLine';
+import CreateTempScale from './CreateTempScale/CreateTempScale';
+import styles from './SvgBackground.module.scss';
+
+const SvgBackground: React.FunctionComponent = () => {
+  const tempUnit = useSelector<CombinedState, TempUnits>(state => state.tempUnit);
+
+  return (
+    <svg height={'400'} width={'1000'} className={styles.svgBackground}>
+      <CreateTempScale
+        coords={{ x: '0', y: '15' }}
+        value={40}
+        tempUnit={tempUnit}
+        className={styles.tempScale}
+      />
+      <CreateTempScale
+        coords={{ x: '0', y: '100' }}
+        value={30}
+        tempUnit={tempUnit}
+        className={styles.tempScale}
+      />
+      <CreateTempScale
+        coords={{ x: '0', y: '200' }}
+        value={20}
+        tempUnit={tempUnit}
+        className={styles.tempScale}
+      />
+      <CreateTempScale
+        coords={{ x: '0', y: '300' }}
+        value={10}
+        tempUnit={tempUnit}
+        className={styles.tempScale}
+      />
+      <CreateTempScale
+        coords={{ x: '0', y: '400' }}
+        value={0}
+        tempUnit={tempUnit}
+        className={styles.tempScale}
+      />
```

Comment:
not sure that creating new function component instead of pure function (as i suggested) helped you to shorten code :)
you can move styles.tempScale to CreateTempScale component and use the classname as default;
the same for color prop in CreateLine

Reply:
also there is no need to pass tempUnit via props, you can just use selector inside CreateTempScale to get it

Reply:
yes, I have noticed that and changed